File: 백준/Bronze/2748. 피보나치 수 2/피보나치 수 2.py
n = int(input())
num_list =[0] * (n+1)
num_list[0] = 0
num_list[1] = 1
fibo = 0;
for i in range(2, n+1):
    num_list[i] = num_list[i-1] + num_list[i-2]

print(num_list[-1])

File: 백준/Bronze/2748. 피보나치 수 2/피보나치 수 2.java
import java.util.Scanner;

public class Main {
	public static void main(String[] args) {
		
		Scanner sc = new Scanner(System.in);
		int fibo = sc.nextInt();
		long[] num_list = new long[fibo+1];
		num_list[0] = 0;
		num_list[1] = 1;
		long result = 0;
		for (int i = 2; i < num_list.length; i++) {
			num_list[i] = num_list[i-1] + num_list[i-2];
		    
		}
		result = num_list[num_list.length-1];
		System.out.println(result);
		
	}

}
